### ISSUE TITLE
[buttons & link buttons]: transitions are missing. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1719]
I just noticed that buttons and link buttons no longer have the transition like they used to do before i.e the colour changes instantly without the smoothing. See the below video it shows both the current state of the buttons + the old one with the hover. 

https://user-images.githubusercontent.com/46004116/157040456-cd8ac24d-784b-49c0-97fb-11e49d93d084.mov

**Comment by Cahllagerfeld** [comment]
What was the transition-duration before the change, I assume `200ms` right?. I just ahd a look at the coding and technically there is a duration from currently `50ms` though

**Comment by nisarhassan12** [comment]
Thanks, @Cahllagerfeld for looking into it. 

You're right it was 200ms. Please not that it had a delay as well and that is the transition triggers after 50ms. 

```scss
a {
  transition: all 0.2s 50ms;
}
```

> I just ahd a look at the coding and technically there is a duration from currently 50ms though

I didn't realize that probably because it happens too fast.
